lnrpc/routerrpc/router_server_test.go

```diff
@@ -284,6 +294,7 @@ func TestIsLsp(t *testing.T) {
 		isLsp          bool
 		expectedHints  [][]zpay32.HopHint
 		expectedLspHop *zpay32.HopHint
+		channelID      uint64
```

Comment:
Why is `channelID` defined? It appears to not been used anywhere.

Reply:
removed